test/integration/src/main/java/io/pravega/test/integration/utils/TestUtils.java

```diff
@@ -0,0 +1,214 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.integration.utils;
+
+import com.google.common.base.Preconditions;
+import io.pravega.client.ClientConfig;
+import io.pravega.client.EventStreamClientFactory;
+import io.pravega.client.admin.ReaderGroupManager;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.EventStreamReader;
+import io.pravega.client.stream.EventStreamWriter;
+import io.pravega.client.stream.EventWriterConfig;
+import io.pravega.client.stream.ReaderConfig;
+import io.pravega.client.stream.ReaderGroupConfig;
+import io.pravega.client.stream.ScalingPolicy;
+import io.pravega.client.stream.Stream;
+import io.pravega.client.stream.StreamConfiguration;
+import io.pravega.client.stream.impl.JavaSerializer;
+import io.pravega.common.concurrent.Futures;
+import io.pravega.shared.security.auth.PasswordAuthHandlerInput;
+import io.pravega.shared.security.crypto.StrongPasswordProcessor;
+import lombok.Cleanup;
+import lombok.NonNull;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * A helper class with general-purpose utility methods for integration tests.
+ */
+@Slf4j
+public class TestUtils {
+
+    /**
+     * Creates the specified {@code scope} and {@code streams}, using the specified {@code clientConfig}.
+     *
+     * Note: This method creates the streams using a scaling policy with a fixed number of segments (one each).
+     *
+     * @param clientConfig the {@link ClientConfig} to use for connecting to the server
+     * @param scope the scope
+     * @param streams the streams
+     */
+    public static void createScopeAndStreams(ClientConfig clientConfig, String scope, List<String> streams) {
+        @Cleanup
+        StreamManager streamManager = StreamManager.create(clientConfig);
+        assertNotNull(streamManager);
+
+        boolean isScopeCreated = streamManager.createScope(scope);
+        assertTrue("Failed to create scope", isScopeCreated);
+
+        streams.forEach(s -> {
+            boolean isStreamCreated = streamManager.createStream(scope, s,
+                    StreamConfiguration.builder().scalingPolicy(ScalingPolicy.fixed(1)).build());
+            if (!isStreamCreated) {
```

Comment:
false is returned when the stream already exists. 
so in the idempotent case, do you want to throw the exception from here or return the boolean from this method signature to let the caller know so that they can deal with it.

Reply:
Sure, I've modified the implementation to return a boolean that indicates whether all the objects were newly created or not.